staging/src/k8s.io/apiserver/pkg/registry/rest/rest.go

```diff
@@ -171,6 +171,10 @@ type Creater interface {
 
 	// Create creates a new version of a resource.
 	Create(ctx genericapirequest.Context, obj runtime.Object) (runtime.Object, error)
+
+	// CreateInitialized creates a new version of a resource and waits until the object has been
+	// successfully initialized.
```

Comment:
Add: "Callers should check the returned object to verify that it was completely initialized before returning it to clients."